vmrunner/vmrunner.py

```diff
@@ -328,6 +325,28 @@ def writeline(self, line):
     def poll(self):
         return self._proc.poll()
 
+class solo5_hvt(solo5):
+    def __init__(self, config):
+        super(solo5_hvt, self).__init__(config)
+
+    def name(self):
+        return "Solo5-hvt"
+
+    def boot(self, multiboot, debug=False, kernel_args = "", image_name = None):
+        solo5_bin = INCLUDEOS_HOME + "/x86_64/bin/solo5-hvt"
+        super(solo5_hvt, self).boot(solo5_bin, multiboot, debug, kernel_args, image_name)
+
+class solo5_spt(solo5):
+    def __init__(self, config):
+        super(solo5_spt, self).__init__(config)
+
+    def name(self):
+        return "Solo5-spt"
+
+    def boot(self, multiboot, debug=False, kernel_args = "", image_name = None):
+        solo5_bin = INCLUDEOS_HOME + "x86_64/bin/solo5-spt"
```

Comment:
There is a missing "/" before x86_64

Reply:
I thought I fixed that already.. let me check

Reply:
Yes, its fixed in one of my commits

Reply:
I thought that was fixed for hvt and not spt